components/scream/scripts/scripts-tests

```diff
@@ -409,7 +409,10 @@ class TestTestAllScream(TestBaseOuter.TestBase):
         else:
             self.skipTest("Skipping full run")
 
-    def test_jenkins_process(self):
+    def test_mpirank_and_thread_spreading(self):
```

Comment:
What's the point of this one? Are we putting it there as a placeholder, with the idea of expanding it in a subsequent PR?

Reply:
> Add stubs for mpirank/thread spreading. @bartgol will fill these in once his work is done.

Reply:
lol, I really need to learn to read the PR message before starting the review... :)